pom.xml

```diff
@@ -941,7 +941,8 @@
                         <configuration>
                             <source>8</source>
```

Comment:
Is this version of Java? Why we will ignore failure while generating javadoc?

Reply:
The maven-javadoc-plugin has a regression when compiling with java 11 and fails with unknown error during deploy to nexus. The solution, which the community proposes, is to use java 8, which worked for us quite a long time. However, during release procedure, the javadoc generation failed again and again even after trying numerous combinations of configurations. Me and Sunny decided on setting `<failOnError>false</failOnError>` for now because javadoc is not vital part of our release.

Reply:
okay

Reply:
Maybe some comment should be added with reason why this is configured like that. Probably in future we should delete this configuration.

Reply:
Yes, for sure. This is temp workaround to execute the deployment to nexus.